src/components/Lobby/index.tsx

```diff
@@ -151,10 +152,14 @@ const LobbyScreen = ({ onStartGame }: LobbyScreenProps) => {
                   <Input
                     id="seed"
                     type="text"
+                    ref={seedInputRef}
                     className="w-auto rounded-none border-0 border-transparent border-white bg-transparent text-end focus-visible:border-b-[1px] focus-visible:ring-0 focus-visible:ring-offset-0"
                     value={seed}
                     onChange={(e) => setSeed(e.target.value)}
                   />
+                  <Button size="icon" variant="tertiary" onClick={() => seedInputRef?.current?.focus()}>
+                    <FaPen />
+                  </Button>
```

Comment:
Would also make sense to select whole text

```suggestion
                  <Button size="icon" variant="tertiary" onClick={() => {
                    seedInputRef?.current?.focus();
                    seedInputRef?.current?.select();
                  }}>
                    <FaPen />
                  </Button>
```